### DISCUSSION TITLE
Welcome to Ansible Real Life discussion forum

### DISCUSSION THREAD

**Discussion by iamgini** [iamgini/ansible-real-life#2] [Announcements]
Announcing Discussion forum for **[ansible-real-life](https://github.com/ginigangadharan/ansible-real-life)** !

You can add discussions here such as,
- Requests
- Ideas
- Questions on use cases
- and so on.

Quick issues, please join our [telegram group - t.me/techbeatly](https://t.me/techbeatly) and ask questions there.

Happy Learning

**Comment by amaurybsouza** [comment]
hi @ginigangadharan, I would like to add some ideas about Ansible. I have worked with Ansible for a couple of years and I am able to contribute with more details around Ansible.

Amaury.

**Reply by iamgini** [reply]
@amaurybsouza wonderful!
please proceed.

you can raise by ideas or raise PR by following the structure of the repo

And if you are interested:
https://www.techbeatly.com/contribute/

Thank you.

**Comment by altaf1203** [comment]
How to perform mvn clean install from controller node to destination nodes. I have tried with command module with args , chdir and environment module
